repos/spack_repo/builtin/packages/libmetatensor/package.py

```diff
@@ -14,16 +14,19 @@ class Libmetatensor(CMakePackage):
     url = "https://github.com/metatensor/metatensor/releases/download/metatensor-core-v0.0.0/metatensor-core-cxx-0.0.0.tar.gz"
     git = "https://github.com/metatensor/metatensor.git"
 
-    maintainers("HaoZeke", "luthaf", "rmeli")
+    maintainers("HaoZeke", "Luthaf", "RMeli")
     license("BSD-3-Clause", checked_by="HaoZeke")
 
+    version("0.1.19", sha256="2d319186057cf6da8fe39cc4f961baccce59c4486223113ce554632ae7765e26")
+    version("0.1.18", sha256="617680f86751826221f7738f30e66cda350dd8f79725aba2b0c25df91d0d8614")
```

Comment:
This one was buggy and yanked from other package managers, so maybe it should also be excluded from spack?